schema/test/unit/functions/funding_form_changes_to_separate_schemas_test.sql

```diff
@@ -43,13 +79,13 @@ select is (
 
 select is(
   (select count(*) from cif.form_change where json_schema_name='funding_parameter_EP'),
-  2::bigint,
+  3::bigint,
   'All EP projects have had their funding_parameter json_schema_name changed to funding_parameter_EP'
 );
 
 select is(
   (select count(*) from cif.form_change where json_schema_name='funding_parameter_IA'),
-  3::bigint,
+  2::bigint,
   'All IA projects have had their funding_parameter json_schema_name changed to funding_parameter_IA'
 );
 
```

Comment:
One more test needed here now that I think of it. We should ensure that no other form_change records were changed as a result of this function.